### ISSUE TITLE
Use es6 import / export in WebPack

### ISSUE THREAD

**Issue by arxpoetica** [sveltejs/sapper-template#66]
What would it take to use import instead of require?

**Comment by Rich-Harris** [comment]
Sapper would need to use https://github.com/standard-things/esm, basically. I'm not sure it's justified though — is it really a pain point?

**Comment by arxpoetica** [comment]
For my specific use case, yes. I have a config file that drives dotenv with a bunch of specialized settings, but I had to duplicate it exactly w requires for webpack. So now I have two duplicate config files for es6 and older require...which I have to keep parity for.

**Comment by Rich-Harris** [comment]
I don't think I understand, can you ELI5?

**Comment by arxpoetica** [comment]
If I have shared settings or config or other files between the main Sapper app and the webpack config files, it would be nice to use one common es6 `import/export` language, and not have to duplicate files to meet two standards.

I currently have two config.js files, one for webpack with `require` statements and one for the main Sapper app with `import/export` statements. Those two files are in essence exactly the same. I'd like to make one file to rule them all.

**Comment by Rich-Harris** [comment]
In that case I'd suggest using https://github.com/standard-things/esm, and loading that file inside your webpack config with

```js
// inside webpack config
require = require('esm')(module);
const config = require('../config.js');
```

**Comment by arxpoetica** [comment]
Huh. Already had esm installed for another thingz, but hadn't realized I could **just use it**™ out of the box like that for webpack.

I'll give it a try and close this ticket.

**Comment by Rich-Harris** [comment]
@arxpoetica we good to close this?

**Comment by arxpoetica** [comment]
I never got around to testing it, but I'll reopen if I run into bumps.
